tests/serial/convergence-measures/testConvergenceMeasures2.xml

```diff
@@ -44,12 +44,7 @@
     <time-window-size value="1.0" />
     <exchange data="Data1" mesh="MeshOne" from="SolverOne" to="SolverTwo" />
     <exchange data="Data2" mesh="MeshOne" from="SolverTwo" to="SolverOne" />
-    <max-iterations value="100" />
-    <absolute-convergence-measure limit="1e-1" data="Data2" mesh="MeshOne" />
-    <min-iteration-convergence-measure
-      min-iterations="3"
-      data="Data1"
-      mesh="MeshOne"
-      suffices="true" />
+    <max-iterations value="3" />
+    <absolute-convergence-measure limit="1e-3" data="Data2" mesh="MeshOne" />
```

Comment:
```suggestion
    <absolute-convergence-measure limit="1e-3" data="Data2" mesh="MeshOne" />
    <min-iterations value="3" />
```

Reply:
The min-iter-c-m `suffices`, so this always only iterates to 3 iterations.
Again maybe that was the point here. #1846